### ISSUE TITLE
Use the Travis configuration to start the test MongoDB instance.

### ISSUE THREAD

**Issue by allanbank** [brianfrankcooper/YCSB#284]
The MongoDB test cases need a MongoDB instance to connect to. Currently they are using the [Embedmongo.flapdoodle.de](http://flapdoodle-oss.github.io/de.flapdoodle.embed.mongo/) jar to start the MongoDB instance. 

This works fine but leaves copies the MongoDB distribution in the user's home directory which is at a minimum surprising if not obnoxious. 

Instead of starting a MongoDB instance manually the tests should instead test for the existence of a process running on the default MongoDB port and only run the tests if the connection is successful. The travis configuration can be changed to start the MongoDB service for the automated tests.

**Comment by busbey** [comment]
Yes, this is a great idea. Could we also update the current test thing to keep the mongo instance either in the maven build dir or the junit temp space?

**Comment by allanbank** [comment]
@busbey - I tried to keep everything in YCSB/mongodb/target/mongodb directory but I could not figure out how to keep it from dropping the distribution in ${HOME}/.embed* directory. It adds 30 seconds to the builds too so I just punted.  (I honestly don't know Embedmongo.flapdoodle.de very well so is probably me not them.)

Those doing development on the client will know how to start a MongoDB process so it is not that big a deal (and it is in the README.md). With the Travis change it will start one for the automated tests.
